### DISCUSSION TITLE
msys2@2023-07-18: decompress error

### DISCUSSION THREAD

**Discussion by wention** [ScoopInstaller/Main#5187] [Q&A]
7-Zip 23.01 (x64) : Copyright (c) 1999-2023 Igor Pavlov : 2023-06-20

Scanning the drive for archives:
1 file, 57700740 bytes (56 MiB)

Extracting archive: C:\Users\wention\scoop\apps\msys2\2023-07-18\msys2-base-x86_64-20230718.tar.xz
--
Path = C:\Users\wention\scoop\apps\msys2\2023-07-18\msys2-base-x86_64-20230718.tar.xz
Type = xz
Physical Size = 57700740
Method = LZMA2:26 CRC64
Streams = 1
Blocks = 2
Cluster Size = 201326592
Characteristics = BlockPackSize BlockUnpackSize

Sub items Errors: 1

Archives with Errors: 1

Sub items Errors: 1
ERROR: Unexpected end of data : msys2-base-x86_64-20230718.tar

**Comment by rashil2000** [comment]
Could you try again after running `scoop cache rm msys2`?

**Comment by wention** [comment]
it work! sorry about the issue!!
